gulpfile.babel.js

```diff
@@ -19,6 +19,8 @@ import pkg from './package.json';
 
 const dirs = pkg['h5bp-configs'].directories;
 
+const jQueryVersion = pkg.devDependencies.jquery.replace("^","");
```

Comment:
This will have to be considered in the test as well. That's why Travis is failing right now.

https://github.com/h5bp/html5-boilerplate/blob/master/test/file_content.js#L61

Since the version number can now be unknown, the test would have to get a little creative.